.github/workflows/release-and-publish.yml

```diff
@@ -0,0 +1,57 @@
+name: Release and publish
+
+on:
+  push:
+    branches:
+      - main
+
+permissions: {}
+
+jobs:
+  release-please:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: write # to create release commit
+      pull-requests: write # to create release PR
+
+    outputs:
+      release_created: ${{ steps.release.outputs.release_created }}
+
+    steps:
+      - name: 🆕 Create or update release
+        uses: google-github-actions/release-please-action@a37ac6e4f6449ce8b3f7607e4d97d0146028dc0b # 4.1.0
+        id: release
+        with:
+          token: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
consider PAT instead?
>You will want to configure a GitHub Actions secret with a [Personal Access Token](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) if you want GitHub Actions CI checks to run on Release Please PRs.

Reply:
I think there is no need of a PAT as we are giving the gh actions the correct permissions. I tested it in a personal repo: https://github.com/inigomarquinez/howto-release-please/blob/main/.github/workflows/release-and-publish.yml, with no PAT

Reply:
brilliant, in any case we can setup later when needed